Core/include/Acts/Propagator/AtlasStepper.hpp

```diff
@@ -364,7 +364,9 @@ class AtlasStepper {
   }
 
   /// Overstep limit
-  double overstepLimit(const State& /*state*/) const { return m_overstepLimit; }
+  double overstepLimit(const State& /*state*/) const {
+    return -m_overstepLimit;
```

Comment:
I'm not sure I understand what's happening here. Does the sign get interpreted differently now (i.e. against the direction)?

Reply:
I aligned this to the `EigenStepper` impl. we store the `m_overstepLimit` with a positive value and then flip it on return